spring-integration-core/src/main/java/org/springframework/integration/codec/CompositeCodec.java

```diff
@@ -41,9 +39,20 @@ public class CompositeCodec implements Codec {
 
 	public CompositeCodec(Map<Class<?>, Codec> delegates, Codec defaultCodec) {
 		this.defaultCodec = defaultCodec;
+		Assert.notEmpty(delegates, "delegates must not be empty");
 		this.delegates = new HashMap<Class<?>, Codec>(delegates);
 	}
 
+	/**
+	 * @since 7.0
+	 *
```

Comment:
No blank lines in the method JavaDoc.
I think our `src/idea/spring-framework.xml` does the trick to disable JavaDoc reformatting if that is the case.